libbeat/docs/processors-config.asciidoc

```diff
@@ -266,11 +266,12 @@ The `add_cloud_metadata` processor enriches each event with instance metadata
 from the machine's hosting provider. At startup it will detect the hosting
 provider and cache the instance metadata.
 
-Three cloud providers are supported.
+Four cloud providers are supported.
```

Comment:
We can probably drop the number here. More like `The following cloud providers are supported:`